### ISSUE TITLE
server-side-image-map rule should report incomplete

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#408]
server-side-image-map are currently treated as though they are always an accessibility violation. They aren't, so we should just have them go to `incomplete`.

**Comment by jeankaplansky** [comment]
no user-facing docs required.

**Comment by padmavemulapati** [comment]
Verified with latest axe and axe-coconut, 
```
<p><img ismap src="image.jpg"  id="incomplete" /></p>
<p><img src="image.jpg" id="pass" /></p>
```
reporting incomplete

**Comment by jeankaplansky** [comment]
fix
